isso/ext/notifications.py

```diff
@@ -139,17 +139,42 @@ def format(self, thread, comment, parent_comment, recipient=None, admin=False):
 
         else:
             uri = self.public_endpoint + "/id/%i" % parent_comment["id"]
-            key = self.isso.sign(('unsubscribe', recipient))
+            key = self.isso.sign(('unsubscribe', recipient_comment["email"]))
 
-            rv.write("Unsubscribe from this conversation: %s\n" % (uri + "/unsubscribe/" + quote(recipient) + "/" + key))
+            rv.write("Unsubscribe from this conversation: %s\n" % (uri + "/unsubscribe/" + quote(recipient_comment["email"]) + "/" + key))
 
         rv.seek(0)
         return rv.read()
 
+    def notify_subject(self, thread, comment, parent_comment=None, recipient_comment=None, admin=False):
+        if admin:
+            return self.isso.conf.get("mail", "subject_admin").format(
```

Comment:
What happens if subject_admin is not set?

Reply:
The default fallback settings of the options are set in `share/isso.conf` . Refer to that file to check what will happen in that case.